build-system/babel-plugins/babel-plugin-transform-stringish-literals/index.js

```diff
@@ -14,16 +14,9 @@
  * limitations under the License.
  */
 
-const MERGEABLE_TYPES = ['StringLiteral', 'NumericLiteral', 'TemplateLiteral'];
-
 module.exports = function ({types: t}) {
   const cloneNodes = (nodes) => nodes.map((node) => t.cloneNode(node));
-  const escapeStringForTemplateLiteral = (value) =>
-    String(value).replace(/\x60/g, '\\`');
-  const canMergeBinaryExpression = (binaryExpression) =>
-    MERGEABLE_TYPES.includes(binaryExpression.left.type) &&
-    MERGEABLE_TYPES.includes(binaryExpression.right.type) &&
-    binaryExpression.operator === '+';
+  const escapeForLiteral = (value) => String(value).replace(/\x60/g, '\\`');
```

Comment:
super minor nit, could you move the `\x60` to a constant with a name or add a comment

Reply:
Addressed with #fc13643.